packages/grafana-ui/src/components/FlexBox/FlexBox.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { CSSProperties, HTMLAttributes } from 'react';
+import { css, cx } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { useTheme2 } from '../../themes/ThemeContext';
+import { stylesFactory } from '../../themes/stylesFactory';
+
+export interface Props extends HTMLAttributes<HTMLDivElement> {
+  direction?: CSSProperties['flexDirection'];
+  alignItems?: CSSProperties['alignItems'];
+  justifyContent?: CSSProperties['justifyContent'];
+  wrap?: boolean;
+  padding?: number;
+  gap?: number;
+}
```

Comment:
I think if we allowed `className` prop here, it'd make the component really flexible. That's the main issue I have with the current layout component.